app/views/patients/pledge.js.erb

```diff
@@ -0,0 +1,8 @@
+$('.modal-content').html('')
+$('.modal-content').append(
+  '<%= escape_javascript(render partial: "patients/pledge", locals: { patient: @patient, disable_next: disable_continue?(@patient) })%>');
+
+$('.modal').modal('toggle');
+$('.js-title-step span').slice(1).remove();
+
```

Comment:
Here is the issue, new text gets appended to ```.js-title-step```. I don't want to do a direct string check if I can help it. The slice method gets rid of the 1 label number that keeps being appended.